soccer/gameplay/root_play.py

```diff
@@ -192,7 +195,7 @@ def goalie_id(self):
 
     @goalie_id.setter
     def goalie_id(self, value):
-        self._goalie_id = None if value == -1 else value
+        self._goalie_id = value
```

Comment:
@jgkamat I believe this method gets used in the c++ side through the wrapper which makes it pretty hard to track. As for the reason I removed that check, it is because the c++ side now uses None instead of -1 to track if the goalie value is set, see my previous comment for more information.